server/middleware/checkPrisonerInCaseloadMiddleware.test.ts

```diff
@@ -16,7 +16,7 @@ describe('checkPrisonerInCaseloadMiddleware', () => {
   beforeEach(() => {
     jest.resetAllMocks()
     res.locals.user = hmppsUser()
-    res.locals.prisonerSummary = aValidPrisonerSummary(prisonNumber, prisonersPrisonId)
+    res.locals.prisonerSummary = aValidPrisonerSummary({ prisonNumber, prisonId: prisonersPrisonId })
```

Comment:
The vast majority of changes in this PR are actually how we call `aValidPrisonerSummary`
Because of the nature of the main changes in the PR I needed to change the function sig of `aValidPrisonerSummary`, hence it has meant that everywhere we call it I've needed to make this type of change